openedx/features/caliper_tracking/apps.py

```diff
@@ -0,0 +1,22 @@
+
+from django.apps import AppConfig
+from django.conf import settings
+
+
+LOGIN_EVENT_UTIL = 'caliper_tracking.helpers.emit_login_event'
```

Comment:
Please rename this to a more suitable identifier.

Reply:
done. please review now.